pkg/plugin/plugin_test.go

```diff
@@ -210,6 +210,61 @@ var _ = Describe("CNI Plugin", func() {
 				testAddDel(conf, false)
 			})
 		})
+		Context("random mac address on container interface", func() {
+			It("should create eth0 on two different namespace with different mac addresses", func() {
+				conf := fmt.Sprintf(`{
+				"cniVersion": "0.3.1",
+				"name": "mynet",
+				"type": "ovs",
+				"bridge": "%s",
+				"vlan": %d
+				}`, BRIDGE_NAME, VLAN_ID)
+
+				const IFNAME = "eth0"
+
+				By("Creating two temporary target namespace to simulate two containers")
+				targetNsOne, err := ns.NewNS()
+				Expect(err).NotTo(HaveOccurred())
+				defer targetNsOne.Close()
+				targetNsTwo, err := ns.NewNS()
+				Expect(err).NotTo(HaveOccurred())
+				defer targetNsTwo.Close()
+
+				args := &skel.CmdArgs{
+					ContainerID: "dummy",
+					Netns:       targetNsOne.Path(),
+					IfName:      IFNAME,
+					StdinData:   []byte(conf),
+				}
+
+				var result *current.Result
+
+				By("Calling ADD command")
+				r, _, err := cmdAddWithArgs(args, func() error {
```

Comment:
Could you please keep code to call ADD and obtain Interface as inline function and then call it twice, each time for one of the namespaces? To improve readability.

Reply:
Done